uasc/secure_channel.go

```diff
@@ -51,6 +56,12 @@ type SecureChannel struct {
 	// handle which is part of the Request and Response headers.
 	mu      sync.Mutex
 	handler map[uint32]chan Response
+
+	// Encryption Algorithms for sending and receiving messages
+	// During the OpenSecureChannel phase, this contains the Asymmetric Algorithms
+	// After both secure nonces are received from the OpenSecureChannel phase,
+	// this is replaced by the Symmetric algorithm methods for all other communication.
+	encryptionAlgorithm *securitypolicy.EncryptionAlgorithm
```

Comment:
Maybe create a package `crypto` and move the `Keyring` and `EncryptionAlgorithm` in there. Also, I'd like to find a shorter name for `encryptionAlgorithm`. How about `enc`?

Reply:
package crypto is used in the standard library so I'd like to avoid that name (which is used extensively in the securitypolicy package.  Open to alternative ideas.
enc works for the name variable.  I'll change that.